tests/validation/tests/v3_api/test_deploy_clusters.py

```diff
@@ -51,20 +60,99 @@ def test_deploy_rke():
         env_details += cluster.name
         print("Successfully deployed {} with kubernetes version {}".format(
             cluster.name, k8s_version))
-    env_details += "'"
     create_config_file(env_details)
 
 
 @if_not_auto_deploy_eks
 def test_deploy_eks():
     print("Deploying EKS Clusters")
+    global env_details
+    errors = []
+    k8s_versions = [EKS_K8S_VERSIONS[0], EKS_K8S_VERSIONS[-1]]
```

Comment:
So we try only the first and the last version in the list?

Reply:
Yes for the support matrix testing we only want to do the first and last versions for these 3 providers (gke, eks, aks).